py/circuitpy_mpconfig.mk

```diff
@@ -207,6 +213,11 @@ CIRCUITPY_ROTARYIO = 1
 endif
 CFLAGS += -DCIRCUITPY_ROTARYIO=$(CIRCUITPY_ROTARYIO)
 
+ifndef CIRCUITPY_COUNTIO
+CIRCUITPY_COUNTIO = $(CIRCUITPY_FULL_BUILD)
+endif
```

Comment:
```suggestion
CIRCUITPY_COUNTIO ?= $(CIRCUITPY_FULL_BUILD)
```

Reply:
Apologies, thanks for this

Reply:
Why is COUNTIO in here twice? This second time will never apply. (Sorry I didn't ask sooner.)

Reply:
No Problem. Thanks again.